spec/factories/users.rb

```diff
@@ -31,5 +31,9 @@
     factory :contact_centre_team_leader do
       permissions { Array(User::CONTACT_CENTRE_TEAM_LEADER_PERMISSION) }
     end
+
+    factory :pension_wise_api do
```

Comment:
Maybe `_user` prefix because when creating one of these it wouldn't always be obvious:

`create(:pension_wise_api)`